index.html

```diff
@@ -0,0 +1,349 @@
+  
+            <ul class="social-networks about-myself-social-ntks">
+              <li>
+              <a href="#" class="social-network">
+                <i class="fa fa-twitter"></i>
+              </a>
+              </li>                        
+              <li>
+              <a href="#" class="social-network">
+                <i class="fa fa-linkedin"></i>
+              </a>
+              </li>
+              <li>
+              <a href="#" class="social-network">
+                <i class="fa fa-medium"></i>
+              </a>
+              </li>
+              <li>
+              <a href="#" class="social-network">
+                <i class="fa fa-github"></i>
+              </a>
+              </li>
+              <li>
+              <a href="#" class="social-network">
+                              <i class="fa fa-angellist"></i>
+              </a>
+              </li>
+              </ul>                
+          </div>
+          <div class="action">
+            <button type="button" class="btn about-me-btn">Get my resume </button>
+          </div>
+        </div>
+        <ul class="skill-list">
+            <!----<div class="frame-12"> --->
+            <li class="frame-7-languages">
+              <a href="#">Languages</a>
+              <i class='fa fa-chevron-down'></i>
+
+               <!---<ul class="language-list"> ---> 
+              <li>
+                <ul class="language-list">               
+                  <li class="frame-9-10-11">                           
+                      <img src="./images/Ellipse-1.png" alt="Ellipse 1(Javascript)">
+                      <a href="#">Javascript</a>                
+                  </li>   
+                  <li class="frame-9-10-11">                    
+                      <img src="./images/ellipse-2.png" alt="Ellipse 1(HTML)">
+                      <a href="#">HTML</a>                             
+                  </li>                         
+                  <li class="frame-9-10-11">               
+                      <img src="./images/ellipse-3.png" alt="Ellipse 3(CSS)">
+                      <a href="#">CSS</a>                    
+                  </li>
+                </ul>
+              </li>
+            
+            <li class="skill-and-framework">
+              <a href="#">Frameworks</a>
+              <i class='fa fa-chevron-right'></i>
+            </li>
+            <li class="skill-and-framework">
+              <a href="#">Skills</a>
+              <i class='fa fa-chevron-right'></i> 
+            </li>
+        </ul>
+      </div>           
+
+    </section>
+    <!---------End of about myself section-------->
+    <!---------End of mobile-finish-version----->
+
+
+    <!--------Form container mobile version------>
+    <footer>
+      <div class="form-container">
+        <img src="/images/contact-form-shapes-mobile.png" alt="Contact Form Shapes" class="contact-form-shapes">
```

Comment:
- [ ] Kindly note that this image is meant for decoration propose as they add no meaning to the content of the webpage so it will be better if you implement this as a` background-image` in your CSS 👍🏽